packages/flex-plugin-e2e-tests/src/utils/pages/index.ts

```diff
@@ -0,0 +1,96 @@
+import { existsSync, mkdirSync } from 'fs';
+
+import { Page } from 'puppeteer';
+
+import { joinPath } from '..';
+import { AdminDashboard } from './view/admin-dashboard';
+import { AgentDesktop } from './view/agent-desktop';
+import { Plugins } from './view/plugins';
+import { TwilioConsole } from './view/twilio-console';
+
+const screenshotExtension = '.png';
+
+export interface BaseUrl {
+  flex: string;
+  twilioConsole: string;
+}
+
+export class App {
```

Comment:
Should this extend Base too?

Reply:
It shouldn't. App serves as an entry point to any page. Essentially, it means that browser is open (and we don't care which view/page it is on. We only define _view_ agnostic functions on it. (Although for `getFlexAccountSid` we do need to be on Flex for it to work).